testing/multi-env-test-engine/src/main/java/org/projectnessie/junit/engine/MultiEnvTestEngine.java

```diff
@@ -88,6 +81,12 @@ public TestDescriptor discover(EngineDiscoveryRequest discoveryRequest, UniqueId
             }
           });
 
+      // JupiterEngineDescriptor must be the root, that's what the JUnit Jupiter engine
+      // implementation expects.
+      JupiterEngineDescriptor multiEnvDescriptor =
+          new JupiterEngineDescriptor(
+              uniqueId, new DefaultJupiterConfiguration(new EmptyConfigurationParameters()));
```

Comment:
Why not use `discoveryRequest.getConfigurationParameters()`?

Reply:
This umbrella descriptor doesn't need configuration at all. The only thing we do with it is call `.addChild()`. We don't actually use it to collect any discovered tests using JUnit's discovery mechanism.

Reply:
What about engine selectors? (if we're running from JUnit 5 Runner CLI... hypothetically)

Reply:
I think it wouldn't make any difference, but I don't mind putting back `discoveryRequest.getConfigurationParameters()` 👍

Reply:
thx!